app/views/foods/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Foods</h1>
+
+<h3>Count of search results: <%= @total_results %></h3>
```

Comment:
From my original question of efficiency, where we may return `total_results` alongside the `food` data array, we could also call to the `@facade` here, where depending on the result we may be able to call `@facade.{some method}.total_hits` or something to that effect. Not the end of the world but in a costly API call, this can save a lot of time on page render.